### PR TITLE
Add course to courses\free-courses-en.md

### PR DESCRIPTION
**Pull request by cireneirbo**
## What does this PR do?
Add resource to courses

### Description
A free, non-profit bootcamp's curriculum for learning web development.

### Why is this valuable (or not)?
It teaches a variety of skills, has many projects to work on, and can help bring a beginner up to a level where they can look for a job as a web developer.

### How do we know it's really free?
It has a creative commons license (https://education.launchcode.org/intro-to-professional-web-dev/appendices/about-this-book.html)

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course, as there is a significant number of assignments and free resources available to learn through.

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [X] Search for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by davorpa** [comment]
**TIP :rocket::** Thanks for taking your time on fill Pull Request fields in your first contribution :heart: 

It's always a good practice which it use to increase the mantainers productivity.

**Review by davorpa — CHANGES_REQUESTED** [review]
Both are books

**Comment by cireneirbo** [comment]
I'll make some changes! To me it was a course, but you're right - definitely books

**Comment by cireneirbo** [comment]
whoops! I need to delete the courses! One sec and I'll resend

**Comment by eshellman** [comment]
I looked a bit closer at these and noticed that many of the chapters include videos. So maybe "course" is a better category. (@cireneirbo sorry to not look more closely sooner; now that Hacktoberfest is over we have more time for thoughtful consideration.) We weight you opinion highly - compare these resources to others in books and in courses.

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1031,6 +1031,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Java Programming](https://en.wikibooks.org/wiki/Java_Programming) - Wikibooks
 * [Java Programming for Kids](https://yfain.github.io/Java4Kids/) - Yakov Fain
 * [Java Projects, Second Edition](https://www.packtpub.com/free-ebooks/java-projects-second-edition) - Peter Verhas (Packt account *required*)
+* [Java Web Development](https://education.launchcode.org/java-web-development/) - Chris Bay, Jim Flores, Carly Langlois, Sally Steuterman (The LaunchCode Foundation)
```
Comment (davorpa) [inline books/free-programming-books-langs.md:1034]: As Eric says... 

Although most is text and is called itself "book", it has some videos that cant' replace text. E.g:

https://education.launchcode.org/java-web-development/chapters/orm-part1/background.html

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -1115,6 +1116,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Exploring ES6](http://exploringjs.com/es6/) - Dr. Axel Rauschmayer (HTML)
 * [Google JavaScript Style Guide](https://google.github.io/styleguide/javascriptguide.xml) - Aaron Whyte, Bob Jervis, Dan Pupius, Erik Arvidsson, Fritz Schneider, & Robby Walker (HTML)
 * [Human JavaScript](http://read.humanjavascript.com/ch01-introduction.html) - Henrik Joreteg (HTML)
+* [Introduction to Professional Web Development in JavaScript](https://education.launchcode.org/intro-to-professional-web-dev/) - Chris Bay, Jim Flores, Blake Mills, Sally Steuterman, Paul Matthews, Carly Langlois (The LaunchCode Foundation)
```
Comment (davorpa) [inline books/free-programming-books-langs.md:1119]: More or less the same... It has some non-printable content like gif / animations. E.g:

https://education.launchcode.org/intro-to-professional-web-dev/chapters/angular-lsn3/studio.html

So.. you are right at first time.

**Review by davorpa — APPROVED** [review]
LGTM :+1:. Thanks for the effort

File: books/free-programming-books-langs.md (hunk 2 above)
Comment (davorpa) [reply books/free-programming-books-langs.md]: Resolved in https://github.com/EbookFoundation/free-programming-books/pull/6540/commits/106eb8583651375a54400e43261e4d2930b5d4df

File: books/free-programming-books-langs.md (hunk 1 above)
Comment (davorpa) [reply books/free-programming-books-langs.md]: Resolved in https://github.com/EbookFoundation/free-programming-books/pull/6540/commits/106eb8583651375a54400e43261e4d2930b5d4df

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books! And thanks for tolerating the back and forth, it's essential for developing consensus.

**Pull request merged**